kubernetes-model-generator/openshift-model/src/generated/java/io/fabric8/openshift/api/model/Identity.java

```diff
@@ -59,7 +58,7 @@
 @VelocityTransformations({
     @VelocityTransformation(value = "/manifest.vm", outputPath = "openshift.properties", gather = true)
 })
-public class Identity implements HasMetadata, Namespaced
```

Comment:
How come this one changed? I can't see any modifications in existing go code

Reply:
see `kubernetes-model-generator/pkg/schemagen/generate.go` . I've added it to Cluster Scoped resources list

Reply:
Nevermind, saw the change now, the GitHub file filter is not working well.

I assume this is correcting a previous misconfiguration.